### PR TITLE
Link to Apollo17.org.

### PR DESCRIPTION
**Pull request by jaylett**

### CODE REVIEW

File: global/templates/base.html [hunk 1]
```diff
@@ -55,6 +55,7 @@
   <ul class='reading-list'>
     <li><a href='http://history.nasa.gov/alsj/'><cite>Apollo Lunar Surface Journal</cite></a> edited by Eric M Jones and Ken Glover</li>
     <li><a href='http://history.nasa.gov/afj/'><cite>Apollo Flight Journal</cite></a> edited by David Woods</li>
+    <li><a href='http://apollo17.org/'><cite>Project Apollo 17</cite></a> built by <a class='author' href='http://benfeist.com/'>Ben Feist</a>, working with the AFJ</li>
```
Comment (georgebrock) [inline global/templates/base.html]: `abbr` element for AFJ?

**Comment by georgebrock** [comment]
Other than the possibly-missing `abbr`, this looks good to me.

File: global/templates/base.html (hunk 1 above)
Comment (jaylett) [inline global/templates/base.html]: We don't higher up, so I'm inclined not to…

Comment (georgebrock) [inline global/templates/base.html]: Fair enough.

**Pull request merged**